source/main.cpp

```diff
@@ -48,6 +51,11 @@ bool enabledNotify = true;
 int offsetHours = 0;
 int offsetMinutes = 0;
 
+static ConfigItemTime *sysTimeHandle;
+static ConfigItemTime *ntpTimeHandle;
+static std::thread *settingsThread;
```

Comment:
Can `std::thread` be a global variable rather than a pointer? I wonder if that makes any difference, maybe it helps with memory management?

Reply:
I'm not that familiar ith C++. Maybe try and tell me? ;)